clang/include/clang/Format/Format.h

```diff
@@ -225,6 +225,21 @@ struct FormatStyle {
     ///   bbb = 2;
     /// \endcode
     bool AlignCompound;
+    /// Only for ``AlignConsecutiveDeclarations``. Whether function declarations
+    /// are aligned.
+    /// \code
+    ///   true:
+    ///   unsigned int f1(void);
+    ///   void         f2(void);
+    ///   size_t       f3(void);
+    ///
+    ///   false:
+    ///   unsigned int f1(void);
+    ///   void f2(void);
+    ///   size_t f3(void);
+    /// \endcode
+    /// \version 20
```

Comment:
Delete.

Reply:
Delete what?  All docs related to the new functionality? Or just the \version which was requested by @mydeveloperday ... I don't understand.

Reply:
GitHub review comments is about the line above unless a range of lines are shown like in https://github.com/llvm/llvm-project/pull/106145/files#r1769761150.

Reply:
ok, so omit the \version is what you're saying ... but @mydeveloperday said it should be versioned.  Is the version annotation in the wrong place perhaps?  Or maybe its just unsupported to version a sub attribute.

Reply:
done, i see where you said sub options are not allowed to be versioned.